examples/multilang-multi-project/proto/lib/google/protobuf/unittest_features.proto

```
@@ -42,83 +48,121 @@ enum EnumFeature {
-  VALUE_LIFETIME_EMPTY_SUPPORT = 3 [ feature_support = {} ];
-  VALUE_LIFETIME_FUTURE = 4
-      [ feature_support.edition_introduced = EDITION_99997_TEST_ONLY ];
-  VALUE_LIFETIME_DEPRECATED = 5 [ feature_support = {
-    edition_deprecated : EDITION_99997_TEST_ONLY
-    deprecation_warning : "Custom feature deprecation warning"
-  } ];
-  VALUE_LIFETIME_REMOVED = 6 [ feature_support = {
-    edition_deprecated : EDITION_2023
-    edition_removed : EDITION_99997_TEST_ONLY
-  } ];
+  VALUE_LIFETIME_SUPPORT = 2 [feature_support = {
```

Buf (1.54.0)

Enum value name "VALUE_LIFETIME_SUPPORT" should be prefixed with "VALUE_LIFETIME_FEATURE_".

(ENUM_VALUE_PREFIX)